sound/soc/sof/intel/hda-dai-ops.c

```diff
@@ -237,15 +237,32 @@ static unsigned int generic_calc_stream_format(struct snd_sof_dev *sdev,
 					       struct snd_pcm_substream *substream,
 					       struct snd_pcm_hw_params *params)
 {
+	struct snd_soc_pcm_runtime *rtd = asoc_substream_to_rtd(substream);
+	struct snd_soc_dai *codec_dai;
 	unsigned int format_val;
+	u32 ch_mask = 0;
+	int num_channels;
+	int j;
```

Comment:
Why use j instead of i ?

Reply:
used codec_dai_id for symmetry with the cpu_dai loop in the first patch